JS-Algorithms/challenges.js

```diff
@@ -95,7 +134,12 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  let temp=1n
```

Comment:
The naming can improve the code readability, it makes easier for the team and new team members to understand the codebase.

```suggestion
  let factorial=1n
```